### PR TITLE
Simplify RNA-seq pipeline.

### PR DESCRIPTION
**Pull request by rhpvorderman**
### Checklist
- [x] Pull request details were added to CHANGELOG.md
- [x] Documentation was updated (if required).

This reduces code complexity a lot. Additionaly the aligning pipeline is not required anymore, which was two files that also had a 100+ lines of code.

### CODE REVIEW

File: sample.wdl [hunk 1]
```diff
@@ -19,42 +24,104 @@ workflow Sample {
...
+                 input:
+                indexFiles = select_first([hisat2Index]),
+                inputR1 = readgroup.R1,
+                inputR2 = readgroup.R2,
+                outputBam = outputDir + "/hisat2/" + sample.id + "-" + readgroup.lib_id + "-" + readgroup.id + ".bam",
+                sample = sample.id,
+                library = readgroup.lib_id,
+                readgroup = readgroup.id,
+                platform = platform,
+                dockerImage = dockerImages["hisat2"]
```
Comment (DavyCats) [inline sample.wdl]: The indentation is off here.

File: sample.wdl [hunk 2]
```diff
@@ -19,42 +24,104 @@ workflow Sample {
...
+            }
+        }
+        # Choose whether to use the STAR or HISAT2 bamfiles for downstream analyses,
+        # star is taken over hisat2.
+        File continuationBam = select_first([star.bamFile, hisat2.bamFile])
+    }
+
+    call samtools.Merge as samtoolsMerge {
```
Comment (DavyCats) [inline sample.wdl]: Is this step still necessary? markduplicates can take multiple BAM files as input.

File: sample.wdl [hunk 3]
```diff
@@ -19,42 +24,104 @@ workflow Sample {
...
+            call star_task.Star as star {
+                input:
+                    inputR1 = [qc.qcRead1],
+                    inputR2 = select_all([qc.qcRead2]),
+                    outFileNamePrefix = outputDir + "/star/" + sample.id + "-" + readgroup.lib_id + "-" + readgroup.id + ".",
+                    outSAMattrRGline = [rgLine],
+                    indexFiles = select_first([starIndex]),
+                    dockerImage = dockerImages["star"]
+            }
+        }
```
Comment (DavyCats) [inline sample.wdl]: Is there an index file produced here? The alignment subworkflow would call `samtools index` after star

Comment (rhpvorderman) [reply sample.wdl]: This is done during the merge step. But I will change the workflow a bit. The way you describe is more efficient.

**Pull request merged**